### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by NikolaevaIrina**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128586641/22.2.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E372)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# How to access a tab control in a Detail View layout
 
+This example demonstrates how to implement a View Controller that accesses layout controls and customize them as required. In particular, the second tab in the tab control should become active when the detail form is opened.
```
Comment (NikolaevaIrina) [inline Readme.md]:
```suggestion
This example implements a View Controller that accesses layout controls and customizes them as required. In particular, the second tab in the tab control should become active when the detail form is opened.

```

File: Readme.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128586641/22.2.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E372)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# How to access a tab control in a Detail View layout
```
Comment (NikolaevaIrina) [inline Readme.md]:
```suggestion
# XAF - How to access a tab control in a Detail View layout

```

File: Readme.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
-<img width="601" alt="Du2Er0SPZV" src="https://user-images.githubusercontent.com/14300209/226934893-008269a8-66b7-4f3e-a04a-d1a000d86e83.png">
+<kbd>![|image](https://user-images.githubusercontent.com/33227868/231761563-86aaa8fa-ebaf-4b20-958e-a4190fafcfa4.png)</kbd>
 
-### ASP.NET Core Blazor
-![image](https://user-images.githubusercontent.com/33227868/231761563-86aaa8fa-ebaf-4b20-958e-a4190fafcfa4.png)
+## Implementation Details
+
+1. This functionality is implemented in the [WinCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Win/Controllers/WinCustomizeTabControlViewController.cs) and [BlazorCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Blazor.Server/Controllers/BlazorCustomizeTabControlViewController.cs) classes, added to the WinForms and Blazor application projects respectively. Copy these classes to the appropriate projects in your test solution.
+2. To identify the layout element by the **TabbedGroup** string, make sure this identifier is specified under the **Views | YourBusinessObject_DetailView | Layout** node in the Model Editor (invoked for the _YourSolutionName.Module/Model.DesignedDiffs.xafml_ file).
```
Comment (AndreyKozhevnikov) [inline Readme.md]: by the **MyTabbedGroup** string

Comment (NikolaevaIrina) [reply Readme.md]:
```suggestion
2. To identify the layout element by the **MyTabbedGroup** string, make sure this identifier is specified under the **Views | YourBusinessObject_DetailView | Layout** node in the Model Editor (invoked for the _YourSolutionName.Module/Model.DesignedDiffs.xafml_ file).

```

File: Readme.md [hunk 4]
```diff
@@ -1,33 +1,37 @@
...
-<p>This example demonstrates how to implement a View Controller that will access layout controls and customize them as required. In particular, the second tab in the tab control should become active when opening the detail form.<br><br></p>
+<kbd>![|image](https://user-images.githubusercontent.com/33227868/231761563-86aaa8fa-ebaf-4b20-958e-a4190fafcfa4.png)</kbd>
 
-### WinForms 
-<img width="601" alt="Du2Er0SPZV" src="https://user-images.githubusercontent.com/14300209/226934893-008269a8-66b7-4f3e-a04a-d1a000d86e83.png">
+## Implementation Details
 
-### ASP.NET Core Blazor
-![image](https://user-images.githubusercontent.com/33227868/231761563-86aaa8fa-ebaf-4b20-958e-a4190fafcfa4.png)
+1. This functionality is implemented in the [WinCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Win/Controllers/WinCustomizeTabControlViewController.cs) and [BlazorCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Blazor.Server/Controllers/BlazorCustomizeTabControlViewController.cs) classes, added to the WinForms and Blazor application projects respectively. Copy these classes to the appropriate projects in your test solution.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
1. This functionality is implemented in the [WinCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Win/Controllers/WinCustomizeTabControlViewController.cs) and [BlazorCustomizeTabControlViewController.cs](./CS/EFCore/AccessLayoutEF/AccessLayoutEF.Blazor.Server/Controllers/BlazorCustomizeTabControlViewController.cs) classes, added to the WinForms and Blazor application projects respectively. Copy these classes to the corresponding projects in your test solution.

```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**